Resources/views/Collector/db.html.twig

```diff
@@ -188,19 +188,40 @@
                 <p>No database queries were performed.</p>
             </div>
         {% else %}
+            {% set group_queries = false %}
+            {% if collector.groupedQueries[connection]|length < queries|length %}
+                {% set group_queries = request.query.has('group') %}
```

Comment:
should be ``request.query.getBoolean('group')``. Otherwise, passing ``?group=false`` would also group them